pkg/sqs/sqs.go

```diff
@@ -150,6 +153,9 @@ func (mh *MessageHandler) triggerDrain(msg *util.Message) {
 	for r := range out.Reservations {
 		for i := range out.Reservations[r].Instances {
 			mh.Drainer.Drain(*out.Reservations[r].Instances[i].PrivateDnsName)
+
+			log.Info("Cooling down %d seconds before moving on...", mh.CoolDown)
+			time.Sleep(time.Duration(mh.CoolDown) * time.Second)
```

Comment:
We just need to be aware, that this countdown gets reset, if node-drainer dies, but I do not see an easy way to fix this.

Reply:
i thought about saving it into a configmap (like in kubernetes-logger) but that felt a bit overpowered for the time being.

Reply:
> but that felt a bit overpowered for the time being.

true